gapic/src/main/com/google/gapid/perfetto/views/FrameEventsSummaryPanel.java

```diff
@@ -323,14 +329,13 @@ private Hover sliceHover(FrameEventsTrack.Data data, Fonts.TextMeasurer m, doubl
         endts = ts + 12;
         ts -= 12;
       }
-      if (data.depths[i] == depth && x >= ts && x<= endts) {
+      if (x >= ts && x<= endts) {
```

Comment:
Same question with comment above.  Is this change necessary?

Reply:
Discussed in our meeting about this and keeping it as is.